src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -129,6 +129,31 @@ private static bool IsExcludedFromCodeCoverage(MethodSymbol method)
             return false;
         }
 
+        private BoundExpression GetFileIndices()
```

Comment:
Fixed - I have made the corresponding function a static / shared  function as we discussed.